candle-transformers/src/models/quantized_t5.rs

```diff
@@ -39,6 +39,34 @@ fn masked_fill(on_false: &Tensor, mask: &Tensor, on_true: f32) -> Result<Tensor>
     Ok(m)
 }
 
+#[derive(Debug, Deserialize, Default, Clone, PartialEq)]
```

Comment:
Rather than duplicating this, use the version from the `t5` module.

Reply:
👍 Done